config/shared/errors/errors.go

```diff
@@ -34,52 +34,54 @@ var (
+	ErrPartitionsOverlap          = errors.New("partitions overlap")
+	ErrPartitionsMisaligned       = errors.New("partitions misaligned")
+	ErrOverwriteAndNilSource      = errors.New("overwrite must be false if source is unspecified")
+	ErrVerificationAndNilSource   = errors.New("source must be specified if verification is specified")
+	ErrFilesystemInvalidFormat    = errors.New("invalid filesystem format")
+	ErrLabelNeedsFormat           = errors.New("filesystem must specify format if label is specified")
+	ErrFormatNilWithOthers        = errors.New("format cannot be empty when path, label, uuid, wipeFilesystem, options, or mountOptions is specified")
+	ErrExt4LabelTooLong           = errors.New("filesystem labels cannot be longer than 16 characters when using ext4")
+	ErrBtrfsLabelTooLong          = errors.New("filesystem labels cannot be longer than 256 characters when using btrfs")
+	ErrXfsLabelTooLong            = errors.New("filesystem labels cannot be longer than 12 characters when using xfs")
+	ErrSwapLabelTooLong           = errors.New("filesystem labels cannot be longer than 15 characters when using swap")
+	ErrVfatLabelTooLong           = errors.New("filesystem labels cannot be longer than 11 characters when using vfat")
+	ErrLuksLabelTooLong           = errors.New("luks device labels cannot be longer than 47 characters")
+	ErrLuksNameContainsSlash      = errors.New("device names cannot contain slashes")
+	ErrInvalidLuksKeyFile         = errors.New("invalid key-file source")
+	ErrClevisPinRequired          = errors.New("missing required custom clevis pin")
+	ErrUnknownClevisPin           = errors.New("unsupported clevis pin")
+	ErrClevisConfigRequired       = errors.New("missing required custom clevis config")
+	ErrClevisCustomWithOthers     = errors.New("cannot use custom clevis config with tpm2, tang, or threshold")
+	ErrTangThumbprintRequired     = errors.New("thumbprint is required")
+	ErrFileIllegalMode            = errors.New("illegal file mode")
+	ErrModeSpecialBits            = errors.New("setuid/setgid/sticky bits are not supported in spec versions older than 3.4.0")
+	ErrBothIDAndNameSet           = errors.New("cannot set both id and name")
+	ErrLabelTooLong               = errors.New("partition labels may not exceed 36 characters")
+	ErrDoesntMatchGUIDRegex       = errors.New("doesn't match the form \"01234567-89AB-CDEF-EDCB-A98765432101\"")
+	ErrLabelContainsColon         = errors.New("partition label will be truncated to text before the colon")
+	ErrNoPath                     = errors.New("path not specified")
+	ErrPathRelative               = errors.New("path not absolute")
+	ErrDirtyPath                  = errors.New("path is not fully simplified")
+	ErrPartitionOverwritten       = errors.New("filesystem overwrites partitioned device")
+	ErrFilesystemNotSetForRewrite = errors.New("disk wipeTable is enabled while wipeFilesystem is not, this will recreate the filesystem every provision")
```

Comment:
Maybe something like:
```suggestion
	ErrFilesystemImplicitWipe = errors.New("device matches disk with wipeTable enabled; filesystem will be wiped")
```